src/endpoints/getResults.ts

```diff
@@ -48,11 +64,21 @@ export const getResults = (config: HLTVConfig) => async ({ pages = 1 } = {}): Pr
           }
 
           const event: Event = {
-            name: matchEl.find('.event-logo').attr('alt'),
-            id: Number(popSlashSource(matchEl.find('.event-logo'))!.split('.')[0])
+            name: eventId ? eventName : matchEl.find('.event-logo').attr('alt'),
```

Comment:
Why not just put the implementation of eventName here? Instead of:

```
let eventName: string

if(eventId) {
  eventName = $('.eventname').text()
}

name: eventId ? eventName : matchEl.find('.event-logo').attr('alt')
```

just

```
name: eventId ? $('.eventname').text() : matchEl.find('.event-logo').attr('alt')
```

Reply:
I didn't want it inside the for loop. This way if you filter by eventid it is resolved only once.

Reply:
OK, so I was on my phone and missed, that I had it insode the for loop. I wanted to set it outside, but for the events it will only get one page, so setting it outside the loop is irrelevant :P
I did it as you suggested.